docs/main/post-install.md

```diff
@@ -18,3 +18,20 @@ Pi-hole will not be used by the host automatically after installation. To have t
 ```code
 static domain_name_servers=127.0.0.1
 ```
+
+## Adding your local user to the 'pihole' group
+
+Since version 6 Pi-hole uses a new API for authentication. All CLI commands use this API instead of e.g. direct database manipulation. If a password is set for API access, the CLI commands also need to authenticate. To avoid entering the password everytime on CLI, Pi-hole allows users which are members of the 'pihole' group to authenicate without manually entering the password (this can be disabled with setting `webserver.api.cli_pw` to `false`.)
```

Comment:
```suggestion
Pi-hole v6 uses a new API for authentication. All CLI commands use this API instead of e.g. direct database manipulation. If a password is set for API access, the CLI commands also need to authenticate. To avoid entering the password everytime on CLI, Pi-hole allows users which are members of the 'pihole' group to authenicate without manually entering the password (this can be disabled by setting `webserver.api.cli_pw` to `false`.)
```